### PR TITLE
Added promise sample

### PR DESCRIPTION
**Pull request by mamaso**

### CODE REVIEW

File: WebJobs.Script.sln [hunk 1]
```diff
@@ -361,6 +361,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ManualTrigger-CSharp", "Man
 		sample\ManualTrigger-CSharp\run.csx = sample\ManualTrigger-CSharp\run.csx
 	EndProjectSection
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "HttpTrigger-Scrape-Promise", "HttpTrigger-Scrape-Promise", "{8E4F0ED5-75E0-4D0C-9469-05F738B1EC38}"
+	ProjectSection(SolutionItems) = preProject
+		sample\HttpTrigger-Scrape-Promise\function.json = sample\HttpTrigger-Scrape-Promise\function.json
```
Comment (brettsam) [inline WebJobs.Script.sln]: Probably name it something that includes `-Node` -- like `HttpTrigger-Promise-Node`.

Comment (mamaso) [inline WebJobs.Script.sln]: yep, scrape prolly has bad connotations anyways :D

Comment (mathewc) [inline WebJobs.Script.sln]: What is "Scrape"?

Comment (mamaso) [inline WebJobs.Script.sln]: Like, web-scraping.  Also, TBD.  To be deleted.

File: sample/HttpTrigger-Scrape-Promise/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿var http = require('http');
+
+module.exports = function (context, req) {
+    // Choose the host from either the body or query string
+    var host = req.body.host || req.query.host;
```
Comment (mathewc) [inline sample/HttpTrigger-Scrape-Promise/index.js]: Does this function forward an arbitrary URL, or just a host?

Perhaps you could do something like take an array of stock names, and return their prices, using `https://www.google.com/finance/info?q=MSFT,GOGL`, and parse/simplify the response to just name/price?

Comment (mamaso) [inline sample/HttpTrigger-Scrape-Promise/index.js]: Yep, just forwards an arbitrary url.  Much cooler idea :)

**Comment by mathewc** [comment]
:ship:

**Pull request merged**